sunflower-land/sunflower-land
# [FEAT] Trade initiates

# Description

Improve UI feedback when a trade has been initiated. When an offer or listing is being fulfilled on chain, display a new informative UI.

<img width="591" alt="Screenshot 2024-12-12 at 8 54 45 AM" src="https://github.com/user-attachments/assets/a7bd8ee9-47ed-4553-8e40-f76cdf707064" />

## How to test?

1. Connect to local API
2. Listing an **on chain** item for sale
3. Open another browser and attempt to purchase (do not submit transaction though)
4. Go back to your initial browser, refresh and try to cancel the listing

_Note: A refresh is needed by the seller as this data is stored on the gameState. Future iterations will try improve this, but it should help with most bugs faced_
